cookbooks/heat_flow/heat-flow-terms.prm

```diff
@@ -0,0 +1,38 @@
+include $ASPECT_SOURCE_DIR/cookbooks/heat_flow/heat-flow.prm
```

Comment:
Maybe worth adding a 1-2 sentences here for those that did not first read the relevant manual section?